packages/backend/src/open_payments/payment_pointer/key/routes.test.ts

```diff
@@ -109,17 +109,5 @@ describe('Payment Pointer Keys Routes', (): void => {
         keys: [jwk]
       })
     })
-
-    test('returns 404 if payment pointer does not exist', async (): Promise<void> => {
```

Comment:
Good question, I don't remember... 🤔

Reply:
I remember now. It was because `paymentPointer` cannot be `undefined` on `PaymentPointerContext`. I'll need to define another context for this to work.

Reply:
But it currently _is_ possible to the payment pointer keys route to get a `ctx` without `paymentPointer`
https://github.com/interledger/rafiki/blob/04698a2d19ef442c9fc481ec005ee3a675066e57/packages/backend/src/open_payments/payment_pointer/middleware.ts#L10-L22
Maybe it needs a `PaymentPointerKeysContext`(?) that has an optional `paymentPointer`